### PR TITLE
stop button implemented

### PR DESCRIPTION
**Pull request by CallumWoon**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -1,16 +1,25 @@
...
+        intervalID = setInterval(() => {
         seconds++;
         clock.textContent = seconds;
     }, 1000);
     isRunning = true;
     }
 });
+
+stopButton.addEventListener("click", () => {
+    if (isRunning === true) {
```
Comment (mattydono) [inline app.js]: is this check really necessary?

File: stopwatch.html [hunk 2]
```diff
@@ -6,7 +6,7 @@
     </head>
     <body>
         <h2 id="clock">0</h2>
-        <button id="startButton">Start</button>
+        <button id="startButton">Start</button><button id="stopButton">Stop</button>
```
Comment (mattydono) [inline stopwatch.html]: nit: these should be on separate lines

**Review by mattydono — CHANGES_REQUESTED** [review]

File: app.js (hunk 1 above)
Comment (CallumWoon) [reply app.js]: I did put thought into whether it needed it or not. I put it there just as an act of diligence but I agree it doesn't really need it, just extra code right?

File: stopwatch.html (hunk 2 above)
Comment (CallumWoon) [reply stopwatch.html]: Fak

File: app.js (hunk 1 above)
Comment (mattydono) [reply app.js]: it's an unnecessary check. Whether it's running or not we can carry out the same function calls with zero side effects.

**Comment by mattydono** [comment]
have you tested this?

**Review by mattydono — APPROVED** [review]

**Pull request merged**